surround/cli.py

```diff
@@ -266,7 +269,7 @@ def parse_tool_args(parsed_args, remote_parser, tool):
         parse_init_args(parsed_args)
 
 def main():
-    logging.disable(logging.CRITICAL)
+    # logging.disable(logging.CRITICAL)
```

Comment:
Can you remove this comment if it isn't needed?

Reply:
Done!